src/components/CollapsedList/CollapsedList.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { CollapsedList } from '.';
+
+const meta = {
+	title: 'Other/Collapsed List',
+	component: CollapsedList,
+	tags: ['autodocs'],
+} satisfies Meta<typeof CollapsedList>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+	args: {
+		list: [
+			{ name: 'item1', logo: null },
```

Comment:
Not a change for this PR: odd that `logo` *has* to be specified, I would expect it to just be an optional parameter...